usecases/objects/manager.go

```diff
@@ -58,6 +58,9 @@ type schemaManager interface {
 		name string, consistency bool,
 	) (*models.Class, error)
 
+	GetConsistentClassCached(ctx context.Context, principal *models.Principal, name string,
```

Comment:
**suggestion:** `GetCachedClass(...)`

Reply:
Yes i'd agree, something can't be both consistent and cached without checking if the cache is consistent which we don't do.

Reply:
makes sense. will change that